kubernetes_platform/python/README.md

```diff
@@ -1,46 +1,33 @@
-# Kubernetes Platform-specific Features
+# Kubeflow Pipelines SDK kfp-kubernetes API Reference
 
-The `kfp-kubernetes` Python library enables authoring [Kubeflow pipelines](https://www.kubeflow.org/docs/components/pipelines/v2/) with Kubernetes-specific features. These features are supported by the [default KFP open source BE](https://github.com/kubeflow/pipelines/tree/master/backend). Specifically, the `kfp-kubernetes` library supports authoring pipelines that use:
+The Kubeflow Pipelines SDK kfp-kubernetes python library (part of the [Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/) project) is an addon to the [Kubeflow Pipelines SDK](https://kubeflow-pipelines.readthedocs.io/) that enables authoring Kubeflow pipelines with Kubernetes-specific features and concepts, such as:
 
 * [Secrets](https://kubernetes.io/docs/concepts/configuration/secret/)
 * [PersistentVolumeClaims](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims)
-* [ImagePullPolicy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy)
+* [ImagePullPolicies](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy)
+* [Ephemeral volumes](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/)
+* [Node selectors](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector)
+* [Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
+* [Labels and annotations](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/object-meta/#ObjectMeta)
+* and more
 
-See the [`kfp-kubernetes` reference documentation](https://kfp-kubernetes.readthedocs.io/).
+Be sure to check out the full [API Reference](https://kfp-kubernetes.readthedocs.io/) for more details.
 
 ## Installation
-The `kfp-kubernetes` package can be installed as a `kfp` SDK extra dependency with `kfp==2.x.x`:
-<!-- TODO: remove --pre when kfp v2 goes to GA -->
+The `kfp-kubernetes` package can be installed as a KFP SDK extra dependency.
 ```sh
-pip install kfp[kubernetes] --pre
+pip install kfp[kubernetes]
 ```
 
 Or installed independently:
 ```sh
 pip install kfp-kubernetes
 ```
 
-## Example usage
-<!-- TODO: test these examples once the BE implementation exists -->
-### Secret: As environment variable
-```python
-from kfp import dsl
-from kfp import kubernetes
+## Getting started
 
-@dsl.component
-def print_secret():
```

Comment:
would be nice to have this example too, in the README, but since it's already covered in kfp-kubernetes documentation [here](https://kfp-kubernetes.readthedocs.io/en/kfp-kubernetes-1.3.0/), and the documentation page is linked in this README, I don't feel strongly about it.

Reply:
done